server/scripts/import-tests/settings.js

```diff
@@ -0,0 +1,7 @@
+const path = require('path');
+
+const gitCloneDirectory = path.resolve(__dirname, 'tmp');
```

Comment:
This also reminds me, could you include `server/scripts/import-tests/tmp` in the `.gitignore`?

I've been guilty in the past of accidentally including it in a `git add .` (but caught it!) while an import was still happening. It's such a quick process that I suppose it's been missed all this time

Reply:
Done!